From 912c4bff1efd118eed74e0d7c12485975d8c0de2 Mon Sep 17 00:00:00 2001
From: Ricardo Salveti <ricardo@opensourcefoundries.com>
Date: Thu, 26 Apr 2018 20:45:37 -0300
Subject: [PATCH 1/5] colibri_imx7: prefer non secure mode by default

Uset can set secure mode by default by setting the bootm_boot_mode env
variable to "sec" if desired.

Signed-off-by: Ricardo Salveti <ricardo@opensourcefoundries.com>
---
 configs/colibri_imx7_defconfig      | 2 +-
 configs/colibri_imx7_emmc_defconfig | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/configs/colibri_imx7_defconfig b/configs/colibri_imx7_defconfig
index 3e1d3c3f3e..b0b0a4ebb0 100644
--- a/configs/colibri_imx7_defconfig
+++ b/configs/colibri_imx7_defconfig
@@ -2,7 +2,7 @@ CONFIG_ARM=y
 CONFIG_ARCH_MX7=y
 CONFIG_TARGET_COLIBRI_IMX7=y
 CONFIG_TARGET_COLIBRI_IMX7_NAND=y
-CONFIG_ARMV7_BOOT_SEC_DEFAULT=y
+# CONFIG_ARMV7_BOOT_SEC_DEFAULT is not set
 CONFIG_IMX_RDC=y
 CONFIG_IMX_BOOTAUX=y
 CONFIG_VIDEO=y
diff --git a/configs/colibri_imx7_emmc_defconfig b/configs/colibri_imx7_emmc_defconfig
index db1de19a8f..8c7182fbc8 100644
--- a/configs/colibri_imx7_emmc_defconfig
+++ b/configs/colibri_imx7_emmc_defconfig
@@ -2,7 +2,7 @@ CONFIG_ARM=y
 CONFIG_ARCH_MX7=y
 CONFIG_TARGET_COLIBRI_IMX7=y
 CONFIG_TARGET_COLIBRI_IMX7_EMMC=y
-CONFIG_ARMV7_BOOT_SEC_DEFAULT=y
+# CONFIG_ARMV7_BOOT_SEC_DEFAULT is not set
 CONFIG_IMX_RDC=y
 CONFIG_IMX_BOOTAUX=y
 CONFIG_VIDEO=y
-- 
2.18.0

